scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/policy/edf/EDFPolicyExtended.java

```diff
@@ -25,17 +25,18 @@
  */
 package org.ow2.proactive.scheduler.policy.edf;
 
+import static java.util.Arrays.asList;
+
 import java.time.Duration;
-import java.util.Calendar;
-import java.util.Comparator;
-import java.util.Date;
-import java.util.LinkedList;
-import java.util.List;
+import java.util.*;
+import java.util.function.Function;
 import java.util.stream.Collectors;
 
 import org.apache.log4j.Logger;
 import org.ow2.proactive.scheduler.common.JobDescriptor;
 import org.ow2.proactive.scheduler.common.job.JobPriority;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.ow2.proactive.scheduler.common.job.JobPriority'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)